src/IssueViz.UnitTests/Selection/AnalysisIssueSelectionServiceTests.cs

```diff
@@ -34,67 +34,90 @@ namespace SonarLint.VisualStudio.IssueVisualization.UnitTests
     [TestClass]
     public class AnalysisIssueSelectionServiceTests
     {
-        private Mock<IVsMonitorSelection> monitorSelectionMock;
-
-        private AnalysisIssueSelectionService testSubject;
+        [TestMethod]
+        public void MefCtor_CheckIsExported()
+        {
+            MefTestHelpers.CheckTypeCanBeImported<AnalysisIssueSelectionService, IAnalysisIssueSelectionService>(null,
+                new[]
+                {
+                    MefTestHelpers.CreateExport<SVsServiceProvider>(Mock.Of<IServiceProvider>()),
+                    MefTestHelpers.CreateExport<IIssueSelectionService>(Mock.Of<IIssueSelectionService>())
+                });
+        }
 
-        [TestInitialize]
```

Comment:
I replaced the TestInitialize with a CreateTestSubject method.
The tests were updated accordingly.